docker/init-db.sql

```diff
@@ -0,0 +1,7 @@
+-- CREATE DATABASE database1;
+-- CREATE DATABASE database2;
+--
+-- GRANT ALL PRIVILEGES ON DATABASE database1 TO admin;
+-- GRANT ALL PRIVILEGES ON DATABASE database2 TO admin;
```

Comment:
one database spot_ease for example with multiple schema, each schema owner by accordingly service

Reply:
in the future, we have to call other service to read data, if use restful api -> implement adapter so that make us dificulty maintain -> grant read access to other modules